### PR TITLE
define connectors deps in groups

### PR DESCRIPTION
**Pull request by ChuckHend**
If we're not working with Redis, we shouldn't have to install its dependencies.

### CODE REVIEW

**Comment by ChuckHend** [comment]
[sc-442193]

File: volley/connectors/__init__.py [hunk 1]
```diff
@@ -1,9 +0,0 @@
-from volley.connectors.confluent import ConfluentKafkaConsumer, ConfluentKafkaProducer
-from volley.connectors.rsmq import RSMQConsumer, RSMQProducer
-
-__all__ = [
-    "ConfluentKafkaConsumer",
-    "ConfluentKafkaProducer",
-    "RSMQConsumer",
-    "RSMQProducer",
-]
```
Comment (ChuckHend) [inline volley/connectors/__init__.py:9]: these were here for convenience,`from volley.connectors import ....` if we leave them where they are then they'll only be imported when they are actually used.

**Review by MeeshCompBio — APPROVED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -10,22 +10,31 @@ packages = [
 
 [[tool.poetry.source]]
 name = "shipt"
-url = "https://pypi.shipt.com/simple/"
+url = "https://github.com/shipt/py-volley"
```
Comment (johnklehm) [inline pyproject.toml:13]: Why the change here?

Comment (johnklehm) [reply pyproject.toml:13]: Oh because it's a public repo now derp.

**Review by johnklehm — APPROVED** [review]
